### ISSUE TITLE
"Failed to change owner & group for wx-config…"

### ISSUE THREAD

**Issue by thinkyhead** [repetier/Repetier-Host#381]
When using the built-in (Sparkle?) **Install Update** option in Repetier-Host the installation fails with the following dialog:

![Install Update Fail](https://user-images.githubusercontent.com/698003/37253120-26985942-24fb-11e8-96d9-e88908b3ee23.png)

**Comment by repetier** [comment]
Have no real clue why this happens to some users. What helps is manually downloading and installing the files.

**Comment by thinkyhead** [comment]
> What helps is manually downloading and installing the files.

Yes, well we are forced to. Good luck on a fix. Maybe it's a bug in Sparkle.

**Comment by repetier** [comment]
At least this command is executed by sparkle. So yes, it I think it comes from there. Strangely on my 2 macs I did not get it, but others did as well.
